make build-linux failed

<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺ 
v                            ✰  Thanks for opening an issue! ✰    
v    Before smashing the submit button please review the template.
v    Please also ensure that this is not a duplicate issue :)  
☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->

## Summary of Bug
`$ make build-linux` is not working with go-cosmwasm

```
--> Download go modules to local cache
LEDGER_ENABLED=false GOOS=linux GOARCH=amd64 make build
make[1]: Entering directory '/go/src/terra'
go build -mod=readonly -tags "netgo" -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=terra -X github.com/cosmos/cosmos-sdk/version.ServerName=terrad -X github.com/cosmos/cosmos-sdk/version.ClientName=terracli -X github.com/cosmos/cosmos-sdk/version.Version=0.3.0-26-gfd2f9d6 -X github.com/cosmos/cosmos-sdk/version.Commit=fd2f9d6f28324d213f77c114e769cd51beef6752 -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo"' -o build/terrad ./cmd/terrad
# github.com/confio/go-cosmwasm/api
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: warning: libdl.so.2, needed by /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so, not found (try using -rpath or -rpath-link)
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: warning: librt.so.1, needed by /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so, not found (try using -rpath or -rpath-link)
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: warning: libpthread.so.0, needed by /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so, not found (try using -rpath or -rpath-link)
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: warning: libc.so.6, needed by /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so, not found (try using -rpath or -rpath-link)
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: warning: ld-linux-x86-64.so.2, needed by /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so, not found (try using -rpath or -rpath-link)
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `memset@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_cond_signal@GLIBC_2.3.2'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `clock_gettime@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `strlen@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_mutexattr_settype@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_condattr_destroy@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_rwlock_tryrdlock@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `mmap@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `getcwd@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_key_create@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `strcmp@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `sysconf@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `getpid@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_rwlock_destroy@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `setjmp@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_mutex_destroy@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_cond_destroy@GLIBC_2.3.2'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `syscall@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `writev@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `sigemptyset@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `sigaction@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `bsearch@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `snprintf@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `open@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_key_delete@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `open64@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `__xpg_strerror_r@GLIBC_2.3.4'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `getpagesize@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `__fxstat64@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_mutex_lock@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `realloc@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_cond_wait@GLIBC_2.3.2'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `calloc@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_mutex_unlock@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `dl_iterate_phdr@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_condattr_init@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_cond_init@GLIBC_2.3.2'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_rwlock_rdlock@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `longjmp@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `malloc@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_mutexattr_init@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `__tls_get_addr@GLIBC_2.3'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `memmove@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_rwlock_unlock@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `lseek@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_mutexattr_destroy@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `__errno_location@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `mkdir@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `abort@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `memcpy@GLIBC_2.14'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `mprotect@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `sched_yield@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_getspecific@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `memcmp@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `munmap@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_mutex_init@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `close@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `__xstat64@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `strrchr@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `__fxstat@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `fcntl@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `getenv@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `bcmp@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_condattr_setclock@GLIBC_2.3.3'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_setspecific@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `dladdr@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `free@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `pthread_rwlock_wrlock@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `memchr@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `write@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `read@GLIBC_2.2.5'
/usr/lib/gcc/x86_64-alpine-linux-musl/9.2.0/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/confio/go-cosmwasm@v0.7.2/api/libgo_cosmwasm.so: undefined reference to `posix_memalign@GLIBC_2.2.5'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:76: build] Error 2
make[1]: Leaving directory '/go/src/terra'
make: *** [Makefile:81: build-linux] Error 2
The command '/bin/sh -c apk add --no-cache $PACKAGES &&     make tools &&     make go-mod-cache &&     make build-linux &&     make install' returned a non-zero code: 2
```

## Version

master

## Steps to Reproduce

`$ make build-linux`

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

--- a/README.md
+++ b/README.md
@@ -17,11 +17,17 @@ This code was forked from the `cosmos/gaia` repository and the majority of the c
 
 The `v0.7.2` release is based on `cosmos-sdk` `v0.38.3` and should be used in any public networks (as that fixed a security issue).
 
+## Supported Systems
+
+The supported systems are limited by the dlls created in [`go-cosmwasm`](https://github.com/CosmWasm/go-cosmwasm). In particular, **we only support MacOS and Linux**. Even for Linux, we only support `glibc 2.12+`. This includes all known supported distributions using glibc (CentOS 7 uses 2.12, obsolete Debian Jessy uses 2.19). **We do not currently support `muslc` systems, such as Alpine Linux**, which is popular in docker distributions.
+
+There is some work done in `go-cosmwasm` to extend support, but it is rather tricky with Rust build flags.
+
 ## Stability
 
 **This is alpha software, do not run on a production system.** Notably, we currently provide **no migration path** not even "dump state and restart" to move to future versions. At **beta** we will begin to offer migrations and better backwards compatibility guarantees.
 
-With the `v0.6.0` tag, we are entering semver. That means anything with `v0.6.x` tags is compatible. We will have a series of minor version updates prior to `v1.0.0`, where we offer strong backwards compatibility guarantees. In particular, work has begun in the `cosmwasm` library on `v0.7`, which will change many internal APIs, in order to allow adding other languages for writing smart contracts. We hope to stabilize much of this well before `v1`, but we are still in the process of learning from real-world use-cases
+With the `v0.6.0` tag, we are entering semver. That means anything with `v0.6.x` tags is compatible with each other, and everything with `v0.7.x` tags is compatible with each other. We will have a series of minor version updates prior to `v1.0.0`, where we offer strong backwards compatibility guarantees. In particular, work has begun in the `cosmwasm` library on `v0.8`, which will change many internal APIs, in order to allow adding other languages for writing smart contracts. We hope to stabilize much of this well before `v1`, but we are still in the process of learning from real-world use-cases
 
 ## Encoding
 

